forms/setupdialog.ui

```diff
@@ -224,6 +239,9 @@ min-width: 200px;
    </item>
    <item>
     <widget class="QStackedWidget" name="wizardStackedWidget">
+     <property name="toolTip">
+      <string>Jump to machine key page</string>
+     </property>
```

Comment:
Why is it set on the wizardStackedWidget, the corresponding radioButton already has a tooltip set?